app/Http/Controllers/BookController.php

```diff
@@ -14,29 +16,33 @@ public function __construct()
 
     public function index()
     {
-        $books = Book::all();
+        $books = Book::with('author', 'bookstores')->get();
+
         return view('books.index', compact('books'));
     }
 
     public function create()
     {
-        return view('books.create');
+        $authors = Author::all();
+
+        return view('books.create', compact('authors'));
     }
 
     public function store(Request $request)
     {
         $request->validate([
-            'title' => ['required'],
-            'author' => ['required'],
-            'isbn' => ['required', 'unique:books'],
-            'cover_image' => ['nullable', 'image', 'mimes:jpg,png,jpeg', 'max:2048'],
+            'title' => 'required|string|max:255',
+            'author_id' => 'required|exists:authors,id', // Yazar ID'sinin geçerli olduğunu kontrol et
+            'isbn' => 'required|unique:books,isbn|max:13',
+            'cover_image' => 'nullable|image|mimes:jpg,png,jpeg|max:2048',
+            'bookstores' => 'required|string', // Kitap satış noktalarının doğrulanması (string olarak)
```

Comment:
anladığım kadarıyla bunu virgül ile ayrıştırarak gönderiyorsun. Bunu daha temiz yapmak için array olarak gönderebilir misin bir de? bookstores array beklesin.